ballerina/Module.md

```diff
@@ -184,14 +184,14 @@ Selenium IDE is a handy tool for finding locators for web elements. Here's how t
 
 3. Start a New Project: Click on Create a New Project and name your project.
 
-<img width="947" alt="new project - side" src="./resources/new project - side.png">
+<img width="947" alt="new project - side" src="https://raw.githubusercontent.com/xlibb/module-selenium/refs/heads/main/ballerina/resources/new-project.png">
```

Comment:
Better to add the links as here
https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/master/ballerina/Module.md?plain=1#L17

```suggestion
 ![Start New Project](https://raw.githubusercontent.com/xlibb/module-selenium/408d78447d684ddd8bbad4ae11f5eff4f84f12bc/ballerina/resources/new-project.png)
```

Reply:
You can use the `Copy Image Address` option and get the image raw link. Once you go to that link, copy the url in the browser which would be in the `raw.githubusercontent.com/..../` format

Reply:
Apply this to other places as well